policies/testing.md

```diff
@@ -0,0 +1,42 @@
+Testing Policy
+==============
+
+This applies to all stable and development branches of the main code repository.
+Except where noted below:
+
+- All Pull Requests adding new functionality to the applications, libraries,
+providers or engines must include suitable tests.
+- All Pull Requests fixing a functional behavior defect in the applications,
+libraries, providers or engines must include a test for that defect.
+
+Pull Requests that do not change the functional behaviour of the applications,
+libraries, providers or engines do not require tests to be added. For example
+the following types of changes do not require tests:
+ - Changes to comments, formatting, internal naming or similar
+ - Fixes for performance
+ - Refactoring
+
+To be explicit the above statement means that tests are not required for changes
+in:
+ - Documentation (including CHANGES.md/NEWS.md)
+ - The test suite
```

Comment:
Hmmm...  we have a test utilities library, and we *do* have a test for it: `test/recipes/01-test_test.t` and `test/test_test.c`...

Reply:
These tests haven't aged well.  There have been a lot of things added to the test framework which aren't covered.

Reply:
The test policy says that tests are not required....but it doesn't prevent us from adding them if we want them anyway.